src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -148,5 +184,78 @@ public Contig apply(String contigString) {
 
     return contigs;
   }
+  
+  /**
+   * Get a list of Contigs representing sharded windows of all genomic regions within the variant set.
```

Comment:
This doesn't load the variants.

It just determines what contigs a variant set has and then also shards them.  It comes into play when you use command line argument `--allReferences` via spark or dataflow.